src/__mocks__/msw/handlers/student/traces.handlers.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  createDeletedTraceIdMock,
+  createMockedTracesViewResponse,
+  invalidTraceId,
+  mockedTracesConfiguration,
+  mockedUnassignedTracesSummary
+} from '@/__mocks__/fixtures/student'
+import {
+  getGetTraceConfigInfoUrl,
+  getGetTracesUnassociatedSummaryUrl,
+  type TraceConfigurationInfo,
+  TraceStatus,
+  type TracesViewResponse,
+  type UnassociatedTracesSummaryDTO
+} from '@/api/avenir-esr'
+import { PageSizes } from '@/config'
+import isNil from 'lodash-es/isNil'
+import { http, HttpResponse } from 'msw'
+
+export const tracesHandlers = [
+  http.get(`*${getGetTracesUnassociatedSummaryUrl()}`, () => {
+    return HttpResponse.json<UnassociatedTracesSummaryDTO>(mockedUnassignedTracesSummary, {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json',
+      }
+    })
+  }),
+  http.delete(`*/me/traces/:traceId`, ({ params }) => {
```

Comment:
```suggestion
  }),
  
  http.delete(`*/me/traces/:traceId`, ({ params }) => {
```

Saut de ligne + pourquoi est-ce que tu mets l'endpoint et pas l'appel d'une méthode définie ? Oubli côté back ? Peut-être mettre un TODO ;)